ui/component/page/index.js

```diff
@@ -1,9 +1,11 @@
 import { connect } from 'react-redux';
+import { makeSelectFileRenderModeForUri } from 'redux/selectors/content';
 import { selectClientSetting } from 'redux/selectors/settings';
 import * as SETTINGS from 'constants/settings';
 import Page from './view';
 
 const select = (state, props) => ({
+  renderMode: makeSelectFileRenderModeForUri(state.content.primaryUri)(state),
```

Comment:
This doesn't feel right, but didn't look too deep.
@saltrafael, can you review it overall?

Reply:
looks fair, I wouldn't call primaryUri like this but yeah should just fix the audio case